Source/WebCore/layout/formattingContexts/inline/InlineItemsBuilder.cpp

```diff
@@ -878,7 +878,6 @@ void InlineItemsBuilder::handleTextContent(const InlineTextBox& inlineTextBox, I
     auto currentPosition = partialContentOffset.value_or(0lu);
     ASSERT(currentPosition <= contentLength);
 
-    {
         auto handleSegmentBreak = [&] {
```

Comment:
This whole block needs to be unindented (blame isn't something we try to preserve fwiw)